web/package.json

```diff
@@ -14,22 +14,23 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.6",
-    "@docusaurus/plugin-google-gtag": "^2.0.0-beta.6",
-    "@docusaurus/preset-classic": "2.0.0-beta.6",
-    "@mdx-js/react": "^1.6.21",
-    "@svgr/webpack": "^5.5.0",
-    "clsx": "^1.1.1",
-    "file-loader": "^6.2.0",
-    "prism-react-renderer": "^1.2.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-modal": "^3.14.3",
-    "url-loader": "^4.1.1"
```

Comment:
Does anyone know why we had `url-loader` and `file-loader` in here? They were causing problems, so I removed them. I did not see any references to them in code. They were also not in the brand new docusaurus test project I setup. Maybe just old cruft?

Reply:
I don't remember it actually, probably some leftover. Ok to remove imo.